google_looker_instance/looker_instance_enterprise_full/main.tf

```
@@ -0,0 +1,93 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# [START looker_google_looker_instance_enterprise_full]
+resource "google_looker_instance" "looker-instance" {
```

```suggestion
resource "google_looker_instance" "default" {
```

Please use `default` as the resource name, unless you have name conflict issues.